### PR TITLE
Todos 6

### PR DESCRIPTION
**Pull request by joeariasc**
- Create a new route with the POST verb to /todo

- Create a handler function and associate it to that route.

- Bind the form to the model

- Then use `tx.Create()` to add the todo to the table

### CODE REVIEW

File: app/actions/todos.go [hunk 1]
```diff
@@ -20,7 +20,20 @@ func Index(c buffalo.Context) error {
 	return c.Render(http.StatusOK, r.HTML("todos/index.plush.html"))
 }
 
-func NewTodo(c buffalo.Context) error  {
+func NewTodo(c buffalo.Context) error {
 	c.Set("todo", models.Todo{})
 	return c.Render(http.StatusOK, r.HTML("todos/new.plush.html"))
 }
+
+func StoreTodo(c buffalo.Context) error {
```
Comment (danoviedo91) [inline app/actions/todos.go]: Let's call this SaveTodo instead

File: app/actions/todos.go [hunk 2]
```diff
@@ -20,7 +20,20 @@ func Index(c buffalo.Context) error {
...
+	tx := c.Value("tx").(*pop.Connection)
+	todo := models.Todo{}
+	todo.IsCompleted = false
```
Comment (danoviedo91) [inline app/actions/todos.go]: When declaring a new Todo, this field is set to false by default.
We can remove this line.

File: app/routes.go [hunk 3]
```diff
@@ -16,5 +16,6 @@ func setRoutes(root *buffalo.App) {
 
 	root.GET("/", actions.Index)
 	root.GET("/todo/new", actions.NewTodo)
+	root.POST("/todo/store", actions.StoreTodo)
```
Comment (danoviedo91) [inline app/routes.go]: This is nice, but our convention is as follows:
root.POST("/todo", actions.SaveTodo)

File: app/templates/todos/new.plush.html [hunk 4]
```diff
@@ -3,7 +3,7 @@
     <div class="card shadow-sm">
         <div class="card-body px-5 py-5">
             <h3 class="card-title font-weight-bold">New Task</h3>
-            <%= form_for(todo, {action: "", method: "POST"}) { %>
+            <%= form_for(todo, {action: todoStorePath(), method: "POST"}) { %>
```
Comment (danoviedo91) [inline app/templates/todos/new.plush.html]: If the route name changes, odds are this will too

File: app/actions/todos.go (hunk 2 above)
Comment (joeariasc) [reply app/actions/todos.go]: I think that I didn't set the field `is_completed` as false by default, do I need create another migration ?

Comment (joeariasc) [reply app/actions/todos.go]: Done!

File: app/actions/todos.go (hunk 1 above)
Comment (joeariasc) [reply app/actions/todos.go]: Done!

**Review by danoviedo91 — APPROVED** [review]

**Pull request merged**